src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -702,6 +702,12 @@ private async Task<(bool, string)> SetTriggersAsync(string content)
                 request.Headers.Add(ScriptConstants.AntaresLogIdHeaderName, requestId);
                 request.Headers.Add("User-Agent", ScriptConstants.FunctionsUserAgent);
                 request.Headers.Add(ScriptConstants.SiteTokenHeaderName, token);
+
+                if (isBackgroundSync)
+                {
+                    request.Headers.Add(ScriptConstants.BackgroundSyncHeader, "true");
```

Comment:
You're adding a header so FE can inspect the payload for differences and skip if the same. However, this host code is already checking hash values on background syncs and skipping the FE sync call if nothing has changed. So background sync won't send an unchanged value.

Reply:
its a good point, may be we need to improve the comparison check here. for example if the actual string is different just due to differences in the order of items in the various lists that are part of the payload, we wont skip the call currently.